cypress/integration/account_settings/theme_color/custom_colors_sidebarStyles_spec.js

```diff
@@ -0,0 +1,441 @@
+
+        // 3. Cick on color window to change color
+        cy.get('.saturation-black').click();
+
+        // 4. Click outside of color modal to remove it from view
+        cy.get('#displaySettingsTitle').click();
+
+        // * Check Sidebar Header Text icon color change
+        cy.get('.color-icon').eq(3).should('have.css', 'background-color', 'rgb(129, 65, 65)');
+
+        // * Check Sidebar Header Text color change
+        cy.get('#accountSettingsTitle').should('have.css', 'color', 'rgb(129, 65, 65)');
+
+        // 5. Save Sidebar Header Text color change
+        cy.get('#saveSetting').click();
+
+        // * Check Sidebar Header Text color change after saving
+        cy.get('#accountSettingsTitle').should('have.css', 'color', 'rgb(129, 65, 65)');
+    });
+
+    it('should change Sidebar Unread Text color and verify color change', () => {
+        // 1. Selecting Theme Edit, Custom Theme, and Sidebar Styles dropdown
+        cy.customColors(0, 'Sidebar Styles');
+
+        // 2. Click Sidebar Unread Text
+        cy.get('.input-group-addon').eq(4).click();
+
+        // 3. Click on color window to change color
+        cy.get('.saturation-black').click();
+
+        // 4. Click outside of color modal to remove it from view
+        cy.get('#displaySettingsTitle').click();
+
+        // * Check Sidebar Unread Text icon color change
+        cy.get('.color-icon').eq(4).should('have.css', 'background-color', 'rgb(129, 65, 65)');
+
+        // 5. Save Sidebar Unread Text color change
+        cy.get('#saveSetting').click();
+
+        // 6. Exit user settings
+        cy.get('#accountSettingsHeader > .close').click();
+
+        // * Check Sidebar Unread Text
+        cy.get('.sidebar-item.unread-title').should('have.css', 'color', 'rgb(129, 65, 65)');
+
+        // 7. Open sidebar dropdown
+        cy.get('#sidebarHeaderDropdownButton').click();
+
+        // 8. Select Account Settings
+        cy.get('#accountSettings').click();
+
+        // 9. Click the Display tab
+        cy.get('#displayButton').click();
+    });
+
+    it('should change Sidebar Text Hover BG color and verify color change', () => {
+        // 1. Selecting Theme Edit, Custom Theme, and Sidebar Styles dropdown
+        cy.customColors(0, 'Sidebar Styles');
+
+        // 2. Click Sidebar Text Hover BG
+        cy.get('.input-group-addon').eq(5).click();
+
+        // 3. Click on color bar to change color
+        cy.get('.hue-horizontal').click();
+
+        // 4. Click outside of color modal to remove it from view
+        cy.get('#displaySettingsTitle').click();
+
+        // * Check Sidebar Text Hover BG color icon change
+        cy.get('.color-icon').eq(5).should('have.css', 'background-color', 'rgb(69, 191, 191)');
+
+        // 5. Save Sidebar Text Hover BG color change
+        cy.get('#saveSetting').click();
+
+        // * CSS hover is not currently implemented for cypress, unable to find workaround. mouseenter/mouseover does not work with css check
+
```

Comment:
I've checked this one and I can't find a workaround as well.  Please delete all commented lines.  Before step 5, I'd suggest just to add
```javascript
// * Check that "sidebarTextHoverBg" is updated
cy.get('#pasteBox').scrollIntoView().should('contain', '"sidebarTextHoverBg":"#45bfbf"');
```

Also, please add above check to other tests.

Reply:
Implemented this check in every test. couldn't identify #pasteBox by its variable name, so i created an id for it in `/components/user_settings/display/user_settings_theme/custom_theme_chooser.`

`<textarea
                    ref='textarea'
                    className='form-control'
                    id='pasteBox'`

Reply:
Oh yes, I missed to mention to assign an ID like `pasteBox`.